ReleaseProcess.md

```diff
@@ -0,0 +1,24 @@
+# Release process for OrderlyWeb
+
+1. Get the version of the OrderlyWeb repo that you want to release locally (typically the latest master branch).  
+1. Tag and build the release log by running 
+   `./scripts/release/make-release.py`
+   This script will warn you if any tickets are merged in that are not "Ready 
+   to Deploy". It will prompt you to push the tag to git and to tag and push Docker images by running the tag-images 
+   script.
+1. Check build has passed in Teamcity
+1. Deploy to the support machine(s), test and deploy to production when ready (see below).
+1. Use `RELEASE_LOG.md` to know which tickets to update to the 'Deployed' status
+
+## Deploying to UAT / Science / Production
+
+1. Connect to the machine:
+    - For uat or science, connect as the vagrant user: `ssh vagrant@support.montagu` (or `ssh support.montagu` and then 
+       `sudo su vagrant && cd`). Then run `./uat.sh` or `./science.sh` which will give you a shell inside the virtual 
+       machine.
+    - For production, `ssh -p 10022 montagu@production.montagu.dide.ic.ac.uk`   
+1. Navigate to `orderly-web-deploy`
```

Comment:
Not quite. We deploy from `montagu-orderly-web`, not `orderly-web`, via the script `./start` (having already run `./setup`) - I need to add a README to that repo! https://github.com/vimc/montagu-orderly-web 

But since the config is checked in to `montagu-orderly-web` I wonder if it should be: update the basic config locally and commit, then go onto UAT/Science/Prod and pull the latest config change and run `./start`

Reply:
https://github.com/vimc/montagu-orderly-web/pull/5/files